mcs/class/Mono.Profiler.Log/Mono.Profiler.Log/LogStreamHeader.cs

```diff
@@ -7,6 +7,8 @@
 namespace Mono.Profiler.Log {
 
 	public sealed class LogStreamHeader {
+		const int MinimumMLPDSupportedVersion = 13;
```

Comment:
Let's just call these `MinVersion`/`MaxVersion`. It's obvious from the context what they refer to. Also, empty line after the `class ... {` as per the Mono coding guidelines.

Reply:
done